app/controllers/models_controller.rb

```diff
@@ -0,0 +1,72 @@
+class ModelsController < ApplicationController
+  before_action :set_model, only: [:show, :edit, :update, :destroy]
+
+  # GET /models
+  # GET /models.json
+  def index
+    @models = Model.all
+  end
+
+  # GET /models/1
+  # GET /models/1.json
+  def show; end
+
+  # GET /models/new
+  def new
+    @model = Model.new
+  end
+
+  # GET /models/1/edit
+  def edit; end
+
+  # POST /models
+  # POST /models.json
+  def create
+    @model = Model.new(model_params)
+
+    respond_to do |format|
+      if @model.save
+        format.html{ redirect_to @model, notice: 'Model was successfully created.' }
+        format.json{render :show, status: :created, location: @model}
+      else
+        format.html{ render :new }
+        format.json{render json: @model.errors, status: :unprocessable_entity}
+      end
+    end
+  end
+
+  # PATCH/PUT /models/1
+  # PATCH/PUT /models/1.json
+  def update
+    respond_to do |format|
+      if @model.update(model_params)
+        format.html{redirect_to @model, notice: "Model was successfully updated."}
+        format.json{render :show, status: :ok, location: @model}
+      else
+        format.html{render :edit}
```

Comment:
Similar code found in models_controller.rb:32, models_controller.rb:46 (mass = 32)